Index: Tasks 25.11.21/Task6.py
===================================================================
diff --git a/Tasks 25.11.21/Task6.py b/Tasks 25.11.21/Task6.py
--- a/Tasks 25.11.21/Task6.py	
+++ b/Tasks 25.11.21/Task6.py	
@@ -1,0 +1,9 @@
+# 6) Написать функцию get_words(), которая подсчитывает количество слов в тексте.
+# (В тексте может быть несколько предложений со знаками пунктуации)
+
+def get_words(line: str = input('Enter string: \n')):
+
+    splitting_string = line.split(' ')
+    return print(('Words: ', len(splitting_string)))
+get_words()
+
